src/CLI/ReturnValueChooser.cs

```diff
@@ -22,7 +22,7 @@ public static int GetReturnValue(IReadOnlyCollection<ScanResults> scanResults, E
                 return BadInputParameters;
 
 #pragma warning disable CA1508
-            if (caughtException != null || scanResults.Any(result => result == null))
+            if (caughtException != null || scanResults == null || scanResults.Any(result => result == null))
```

Comment:
If `scanResults == null` we crash before we get to this check, at Program.cs@87, if `options` is not `null`. 
Can reproduce this by running `AxeWindowsCLI.exe --processname`
Adding a guard on `_scanResultsCollection` at Program.cs@85 results in no error message being displayed in this scenario, but the correct value `BadInputParameters` is returned.

Reply:
Thanks, @peterdur! I added a line of code to handle this scenario and still generate an appropriate output message